constraints/utils.js

```
@@ -7,9 +7,12 @@ export const getStoredConstraints = (c) => c?.filter((c) => c.type == 'stored')
 export const hasStoredConstraints = (f) =>
   getStoredConstraints(f.constraints || []).length > 0
 export const getValidator = (v, n) => v.find((w) => w.validator === n)
-export const applyConstraintToRecord = (c, r, f) => {
+export const applyConstraintToRecord = (c, r, f, d) => {
+  const { config = {} } = f.constraints?.find(
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)